internal/tiltfile2/tiltfile_test.go

```diff
@@ -614,7 +614,24 @@ k8s_resource('foo', 'foo.yaml')
 }
 
 func TestDockerComposeResourceCreation(t *testing.T) {
-	// TODO(maia)
+	f := newFixture(t)
+	defer f.TearDown()
+
+	f.setupFoo()
```

Comment:
this is a buncha k8s-specific setup, i'm concerned it muddies the test. maybe just specifically set up the dc-specific bits you need?